src/main/java/com/eternalcode/combat/config/implementation/PluginConfig.java

```diff
@@ -12,26 +14,59 @@
 
 public class PluginConfig implements ReloadableConfig {
 
-    @Description({ " ", "# Whether the player after entering the server should receive information about the new version of the plugin?" })
-    public boolean receiveUpdates = true;
+    @Description("# Do you want to change the plugin messages?")
+    public Messages messages = new Messages();
```

Comment:
Tak jak pisałem wyżej - uważam, że lepiej jest wydzielić to do osobnej klasy, i zrobić jakieś fajne sekcje